zerver/lib/remote_server.py

```diff
@@ -172,6 +172,23 @@ def build_analytics_data(
     )
 
 
+def get_realms_info_for_push_bouncer() -> List[Dict[str, Any]]:
+    realms = Realm.objects.order_by("id")
+    realm_info_dicts = [
+        dict(
+            id=realm.id,
+            uuid=str(realm.uuid),
+            uuid_owner_secret=realm.uuid_owner_secret,
+            subdomain=realm.subdomain,
+            host=realm.host,
+            uri=realm.uri,
```

Comment:
I think a bit more metadata would probably be prudent; thinking specifically: `deactivated`, `date_created` -- those could be helpful in debugging duplication issues.

Reply:
Added those